fabtests/functional/multi_ep.c

```diff
@@ -473,7 +476,7 @@ static int setup_av_ep(int idx)
 	hints->src_addr = NULL;
 	hints->src_addrlen = 0;
 
-	ret = fi_getinfo(FT_FIVERSION, opts.src_addr, NULL, 0, hints, &fi);
+	ret = fi_getinfo(FT_FIVERSION, NULL, NULL, 0, hints, &fi);
```

Comment:
The original intention could be honoring the user specified source address (`-s` option) but ignoring the port (the `-B` option).  How about keeping the `opts.src_addr` parameter but adding the `FI_SOURCE` flag?